tests/conftest.py

```diff
@@ -173,3 +173,10 @@ def _assert_page_snapshot(page: Page, view_name: str) -> None:
         )
 
     return _assert_page_snapshot
+
+
+@pytest.fixture
+def override_app_style(settings):
+    if settings.INSTALLED_APPS[0] != "test_override_app.auth_style_design":
+        settings.INSTALLED_APPS.insert(0, "test_override_app.auth_style_design")
+    return settings
```

Comment:
```suggestion
```
No need for this to return anything. If a test wants to do more stuff with `settings`, they can request it explicitly, which is more obvious.

Reply:
Fixed :+1: